### ISSUE TITLE
Support T=0K materials to some degree

### ISSUE THREAD

**Issue by tkittel** [mctools/ncrystal#22]
As discussed with @marquezj, we should make it possible for NCrystal to calculate structure factors without Debye-Waller factors (since this is needed for LEAPR). Since the factors become unity at T=0K, one way to do so would be to allow T=0K.

Likely we would only support this for creation of Info objects and possibly only from .ncmat files, actual scatter objects seems a bit unphysical :-)

So we need to set up a test that this works (and fix it if it does not).

**Comment by tkittel** [comment]
At the moment it is possible to set the temperature to a very low number:

```
ess_ncrystal_inspectfile 'Al_sg225.ncmat;dcutoff=2;temp=1e-99K'  -d
```

But:

```
ess_ncrystal_inspectfile 'Al_sg225.ncmat;dcutoff=2;temp=0K'  -d
```
Gives no errors, but acts as if the temperature has not been set.

**Comment by tkittel** [comment]
Actually, as figure 1 in our paper (https://doi.org/10.1016/j.cpc.2019.07.015) shows, mean-squared-displacements do not disappear completely at T=0, so Debye-Waller factors will not become unity. Thus, if the goal is to get the F-squared factors without Debye-Waller factors, we should instead add some flag to NCrystal (similarly to expandhkl) so the initialisation will happen without them.

I expect to make a release with this in a few weeks time.

**Comment by tkittel** [comment]
For simplicity I am implementing this now using a simple environment variable (because this feature is not likely to be generally useful). So run with the env var NCRYSTAL_FILLHKL_FORCEUNITDEBYEWALLERFACTOR set, and all Debye-Waller factors will be set to 1.0.

**Comment by tkittel** [comment]
So @marquezj, this old request is now implemented in NCrystal v2.1.0 :-)
